spec/workers/historical_data_deletion_worker.rb

```diff
@@ -79,8 +79,10 @@ def perform
     end
 
     context "when deleting subscribers" do
-      it "should remove all old deactivated subscribers" do
-        create(:subscriber, deactivated_at: historic_date)
+      it "should remove all historic subscribers" do
```

Comment:
We're missing a test here for a historic subscriber with an active subscription.

Reply:
Good spot, have a test, or two.